python/ray/autoscaler/autoscaler.py

```diff
@@ -64,6 +74,93 @@
 # Max number of nodes to launch at a time.
 MAX_CONCURRENT_LAUNCHES = 10
 
+# Print debug string once every this many seconds
+DEBUG_INTERVAL_S = 5
```

Comment:
These constants `MAX_NUM_FAILURES`, `MAX_CONCURRENT_LAUNCHES `, and `DEBUG_INTERVAL_S ` should go in https://github.com/ray-project/ray/blob/master/src/common/state/ray_config.h. All constants are there to simplify debugging (these kinds of constants often need to be changed during debugging) so that you don't have to track down the constants throughout the whole codebase (these are not considered exposed to users).

If you want I can make this change.

Reply:
Hm how do you access them from Python? I'd prefer to keep Python constants in .py files, since it avoids recompiles.

Reply:
You can see how it's done in #1192, it actually involves a number of files to expose it to Python.

Good point about recompilation. Maybe we should have two files. One for C constants and one for Python constants. What do you think about creating `ray/python/ray/constants.py`?